# ⌬ cdd-toolbox ⌬

Welcome to the ***Computational Drug Discovery Toolbox*** repository! This curated collection provides a comprehensive set of resources for Computational Drug Discovery (CDD) and related fields. Whether you are a researcher, student, or professional in the field of computational chemistry, this toolbox aims to serve as a valuable resource.

## Table of Contents

1. [Natural Compounds Libraries](#natural-compounds-libraries)
2. [Chemical Bioactivity Databases](#chemical-bioactivity-databases)
3. [3D Protein Structures Databases](#3d-protein-structures-databases)
4. [Protein Engineering](#protein-engineering)
5. [Binding Site Detection](#binding-site-detection)
6. [Pharmacophore Screening Tools](#pharmacophore-screening-tools)
7. [Molecular docking](#molecular-docking)
8. [ADMET Prediction](#admet-prediction)
9. [QSAR modeling](#qsar-modeling)
10. [Quantum chemistry](#quantum-chemistry)
11. [Molecular dynamics simulations](#molecular-dynamics-simulations)
12. [Topology Preparation](#topology-preparation)
13. [Normal Mode Analysis for Predicting Protein Motions](#normal-mode-analysis-for-predicting-protein-motions)
14. [Virtual Screening Server for Drug Repurposing](#virtual-screening-server-for-drug-repurposing)
15. [Peptide Design Tools](#peptide-design-tools)
16. [PROTAC Database and Ternary Complex Modelling](#protac-database-and-ternary-complex-modelling)
17. [Machine learning for drug discovery](#machine-learning-for-drug-discovery)
18. [Artificial intelligence for drug discovery](#artificial-intelligence-for-drug-discovery)
19. [Retrosynthesis prediction](#retrosynthesis-prediction)
20. [Miscellaneous tools](#miscellaneous-tools)
21. [Cheminformatics Free Courses](#cheminformatics-free-courses)
22. [Blogs](#blogs)

## Natural Compounds Libraries

- [COCONUT Natural Products](https://coconut.naturalproducts.net/) - A comprehensive database boasting over 400,000 natural products aggregated from more than 50 diverse sources.
- [LOTUS Natural Products](https://lotus.naturalproducts.net/) - Similar to COCONUT, LOTUS focuses on molecular annotations, making it an invaluable resource for sourcing organisms.
- [ZINC15 Natural Products](https://zinc15.docking.org/substances/subsets/natural-products/) - This repository contains a vast collection of over 200,000 commercially available natural compounds, primarily utilized for virtual screening purposes.
- [Collective Molecular Activities of Useful Plants](http://bidd.group/CMAUP/) - This database emphasizes the molecular activities of beneficial plants, aiding in the exploration of their pharmacological potential.
- [Natural Product Activity & Species Source Database (NPASS)](http://bidd.group/NPASS/index.php) - NPASS connects 94,000 natural product activities with their respective species sources, facilitating research in natural product discovery.
- [Cannabis Compound Database](https://cannabisdatabase.ca/) - A specialized database housing information on over 6,000 compounds found in cannabis plants, catering to cannabis-related research endeavors.
- [SuperNatural III](https://bioinf-applied.charite.de/supernatural_3/) - Offering insights into natural compounds and their biological activities, SuperNatural III serves as a valuable resource for researchers in various fields.
- [FooDB](https://foodb.ca/): With a comprehensive compilation of over 70,000 food components, FooDB aids researchers in exploring the chemical composition of diverse foods.
- [AfroDB](http://african-compounds.org/about/afrodb/) - Highlighting more than 4,000 compounds sourced from African medicinal plants, AfroDB significantly contributes to ethnopharmacological studies and drug discovery efforts.
- [Comprehensive Marine Natural Products Database](https://www.cmnpd.org/) - This resource provides extensive information on over 31,000 marine-derived natural products, supporting marine biotechnology and pharmaceutical research.
- [SistematX Secondary Metabolites Database](https://sistematx.ufpb.br/) - Focused on over 8,000 secondary metabolites, SistematX serves as a valuable tool for researchers studying natural product chemistry.
- [Eximed Natural-Product-Based Library](https://eximedlab.com/Screening-Compounds.html) - With more than 5,000 natural product-like compounds tailored for high-throughput screening, this library aids in drug discovery efforts.
- [CoumarinDB](https://yboulaamane.github.io/CoumarinDB/) - Specifically targeting approximately 900 naturally occurring coumarin compounds, CoumarinDB is a specialized resource for researchers in this field.
- [ArtemisiaDB](https://yboulaamane.github.io/ArtemisiaDB/) - Dedicated to compounds from the genus Artemisia, this database offers valuable insights for researchers interested in this plant group.
- [OTAVA Natural Product-Like Library](https://otavachemicals.com/products/compound-libraries-for-hts/natural-product-like-library) - Housing over 1,000 natural product-like compounds designed for high-throughput screening, this library is a valuable resource in drug discovery.
- [BIAdb](https://webs.iiitd.edu.in/raghava/biadb/type.php?tp=natural) - Providing information on bioactive peptides and proteins with therapeutic potential, BIAdb supports research in biomedicine and pharmacology.
- [IMPPAT](https://cb.imsc.res.in/imppat/home) - This comprehensive database offers digitized data from over 100 traditional Indian medicine books, 7000+ research articles, and other sources, making it the most extensive repository of phytochemicals found in Indian medicinal plants.
- [NP-MRD](https://np-mrd.org/natural_products) - Focused on over 280,000 NMR studies, NP-MRD provides a detailed exploration of natural products through NMR spectroscopy.
- [IBS Natural Compounds](https://www.ibscreen.com/natural-compounds) - Offering information on over 60,000 natural compounds, IBS Natural Compounds is a valuable resource for researchers in natural product chemistry and drug discovery.
- [Phytochemicals](https://www.phytochemicals.info/) - This comprehensive resource provides extensive information on phytochemicals, supporting research in plant chemistry and pharmacology.
- [NPACT Database](https://webs.iiitd.edu.in/raghava/npact/index.html) - A database of naturally occurring plant-based anticancer compounds, their activities, and their targets.
- [NaturAr](https://naturar.quimica.unlp.edu.ar/en/) - A natural products database dedicated to the cataloging of the rich biodiversity of Argentina.

## Chemical bioactivity databases  

- [ChEMBL database](https://www.ebi.ac.uk/chembl/) - A large-scale bioactivity database that focuses on bioactive molecules and their targets. It contains information on the binding, functional, and ADMET  properties of drugs.  
- [BindingDB](https://www.bindingdb.org/) - A comprehensive database that provides information on the binding affinities of drugs to their target biomolecules.   
- [PubChem database](https://pubchem.ncbi.nlm.nih.gov/) - Offers information on the biological activities of small molecules, including chemical structures, properties, and bioassay data.   
- [PDBbind database](http://www.pdbbind.org.cn/index.php) - A database focusing on the experimentally measured binding affinities of biomolecular complexes. It includes data on protein-ligand complexes derived from the PDB, providing insights into the structures and energetics of protein-ligand interactions.
- [BRENDA Enzymes database](https://www.brenda-enzymes.org/) - A comprehensive enzyme information system that collects and provides data on enzyme function, structure, and properties. It covers a wide range of enzyme-related information.
- [ExCAPE-DB: ExCAPE chemogenomics database](https://solr.ideaconsult.net/search/excape/) - A chemogenomics database that integrates data on chemical compounds, protein targets, and biological activities.  
- [DrugBank](https://go.drugbank.com/) - Provides comprehensive data on approved and investigational drugs, including details on chemical structures, pharmacology, and therapeutic indications.  
- [ZINC](https://zinc.docking.org/) - A platform for researchers to explore and obtain compounds for various computational and experimental studies.  
- [ChemSpider](http://www.chemspider.com/) - A free chemical database offering information on chemical structures, properties, and associated data.
- [DrugSpaceX](https://drugspacex.simm.ac.cn/) - A database designed to explore chemical and biological spaces related to drug discovery.
- [Therapeutics Data Commons](https://tdcommons.ai/) - An AI foundation for therapeutic science, offering an intuitive interface for various learning tasks in the drug discovery field.  
- [Otava Chemicals](https://www.otavachemicals.com/) - A supplier of screening compounds, building blocks, and custom synthesis services for drug discovery and chemical research.  
- [Vitas-M Laboratory](https://vitasmlab.biz/) - A provider of diverse chemical libraries, including natural product derivatives and proprietary compounds, for high-throughput screening and lead discovery.  

## 3D protein structures databases  

- [RCSB Protein Data Bank](https://www.rcsb.org/) - Global repository for biological macromolecule structures, facilitating research in structural biology and drug discovery.
- [Protein Data Bank in Europe](https://www.ebi.ac.uk/pdbe/) - European counterpart to RCSB PDB, providing deposition, retrieval, and analysis of macromolecular structures.
- [Orientation of proteins in membranes database](https://opm.phar.umich.edu/) - Database focusing on spatial arrangements of integral membrane proteins, aiding understanding of their structural features.
- [UniProt](https://www.uniprot.org/) - Comprehensive resource combining protein sequences, structures, functions, and interactions, with links to 3D structures.
- [InterPro](https://www.ebi.ac.uk/interpro/) - Integrated resource classifying proteins into families, predicting domains, and incorporating information from various protein databases.
- [AlphaFold protein structure database](https://alphafold.ebi.ac.uk/) - Database housing predicted protein structures generated by the AlphaFold deep learning system.
- [Proteopedia](https://proteopedia.org/wiki/index.php/Main_Page) - Collaborative web-based resource with interactive 3D visualizations providing information on protein structure and function.

## Protein engineering

- [DynaMut](https://biosig.lab.uq.edu.au/dynamut/) - A web tool for the analysis and prediction of protein stability changes upon mutation using Normal Mode Analysis.  

## Binding site detection

- [ProteinsPlus](https://proteins.plus/) - A tool designed for the identification and analysis of protein binding sites. It facilitates the exploration of protein-ligand interactions.
- [PrankWeb](https://prankweb.cz/) - A web-based platform specializing in the prediction and analysis of protein binding sites.   
- [CASTp](http://sts.bioe.uic.edu/castp/index.html?2r7g) - A resource for the detection and characterization of protein binding sites. It offers insights into the volume and area of cavities on protein surfaces, contributing to the study of ligand binding and functional sites.  
- [CavityPlus](http://www.pkumdl.cn:8000/cavityplus/index.php#/) - A web server designed for the identification and analysis of protein cavities and binding sites. It provides tools for the characterization of binding pockets, assisting researchers in studying protein-ligand interactions and structure-based drug design.
- [CaverWeb: Identification of Tunnels and Channels in Proteins and Analysis of Ligand Transport](https://loschmidt.chemi.muni.cz/caverweb/) - A tool for identifying tunnels and channels in protein structures. It supports the analysis of ligand transport pathways.


## Pharmacophore screening tools  

- [ZINCPharmer](http://zincpharmer.csb.pitt.edu/) - A web-based pharmacophore screening tool that facilitates the exploration of chemical databases. It aids researchers in identifying potential ligands based on pharmacophoric features.
- [Pharmit](https://pharmit.csb.pitt.edu/) - A pharmacophore modeling and virtual screening platform. It enables users to define and search for pharmacophoric patterns within chemical databases, assisting in the identification of compounds with specific bioactive properties for drug development.
- [PharmMapper](https://www.lilab-ecust.cn/pharmmapper/) - A web server designed for pharmacophore mapping with statistical methods.

## Molecular docking

- [OpenBabel](https://openbabel.org/index.html) - An open-source chemical toolbox designed for the interconversion of chemical file formats. Useful for batch preparing ligands for molecular docking.
- [MGLTools](https://ccsb.scripps.edu/mgltools/) - A software package designed for visualization and analysis of molecular structures including preparation of protein and ligand structures. 
- [AutoDockTools](https://autodocksuite.scripps.edu/adt/) - A GUI for setting up and analyzing molecular docking simulations using the AutoDock suite. It provides a user-friendly environment for preparing input files and visualizing docking results.
- [AutoDock Vina](https://vina.scripps.edu/) - A molecular docking program that efficiently predicts the binding modes of small molecules to target proteins. It utilizes an advanced scoring function and optimization algorithm, making it widely used for virtual screening and drug discovery.
- [EasyDockVina2](https://github.com/S3cr3t-SDN/EasyDockVina2) - A user-friendly tool built on top of AutoDock Vina, streamlining the molecular docking process. It simplifies the setup of docking simulations, making it accessible to users with varying levels of expertise.
- [AutoDock Vina web server](https://durrantlab.pitt.edu/webina/) - A web-based interface for molecular docking simulations. It aids in predicting the binding modes of ligands to target proteins. 
- [Smina](https://github.com/mwojcikowski/smina) - A fork of AutoDock Vina with additional features and optimizations. It focuses on improving scoring and minimization.
- [Gnina](https://github.com/gnina/gnina) - A fork of Smina with integrated support for scoring and optimizing ligands using convolutional neural networks.
- [EasyDock](https://github.com/ci-lab-cz/easydock) - A fully automated pipeline for molecular docking based on Vina with a full support of Smina and Gnina.
- [HADDOCK](https://wenmr.science.uu.nl/haddock2.4/) - A platform used for all types of molecular docking including protein, ligand, peptide, and nucleic acid docking.  

## Molecular interaction visualization

- [PLIP - Protein ligand interaction profiler](https://plip-tool.biotec.tu-dresden.de/plip-web/plip/index) - A tool for profiling interactions between proteins and ligands. It helps analyze and visualize protein-ligand interactions, providing insights into molecular binding mechanisms.  
- [LigPlot+](https://www.ebi.ac.uk/thornton-srv/software/LigPlus/) - A Java-based program that automatically generates  2D ligand-protein interaction diagrams.  
- [Discovery Studio Visualizer](https://discover.3ds.com/discovery-studio-visualizer-download) - A powerful molecular visualization tool that allows users to explore and analyze complex biological and chemical information. It provides an intuitive interface for visualizing molecular structures, protein-ligand interactions, and conducting various analyses.  

## Pharmacokinetics parameters prediction tools 

- [SwissADME](http://www.swissadme.ch/) - A web tool for predicting pharmacokinetic parameters and drug-like properties. It assists researchers in assessing the drug-likeness and pharmacokinetic profile of small molecules. Supports multiple instances.
- [pkCSM](https://biosig.lab.uq.edu.au/pkcsm/) - A platform offering predictions for various physicochemical, pharmacokinetic and toxicity properties with the theory behind each prediction. Suitable for ADMET screening.
- [ADMETlab 2.0](https://admetmesh.scbdd.com/) - A comprehensive tool for predicting absorption, distribution, metabolism, excretion, and toxicity properties of chemical compounds.
- [ProTox-II](https://tox-new.charite.de/protox_II/) - A predictive tool for assessing the toxicity endpoints of drugs.  
- [PreADMET](https://preadmet.webservice.bmdrc.org/) - A web service providing predictions for pharmacokinetic properties. Does not support multiple instances.
- [FAF-Drugs](https://bioserv.rpbs.univ-paris-diderot.fr/services.html) - A program designed to filter extensive compound libraries based on ADMET properties before in silico screening or modeling studies.

## QSAR modeling
- [QSAR Toolbox](https://qsartoolbox.org/) - The Toolbox is a free software for transparent chemical hazard assessment, offering tools for data retrieval, metabolism simulation, and property profiling. It aids in identifying analogues and chemical categories for read-across and trend analysis, filling data gaps.
- [OCHEM](https://ochem.eu/home/show.do) - An online platform offering tools for building QSAR models for predictions of chemical properties.
- [ChemMaster](https://crescent-silico.com/chemmaster/) - A general cheminformatics software used to handle chemical data, in particular for drug design purposes including QSAR modeling.
- [3D-QSAR](https://www.3d-qsar.com/) - A compilation of tools available online for 3D-QSAR modeling.

## Quantum chemistry

- [Gaussian](https://gaussian.com/) - A widely used software suite for electronic structure modeling and quantum chemistry calculations.
- [ORCA](https://orcaforum.kofo.mpg.de/app.php/portal) - A popular computational chemistry software package that is widely used for electronic structure calculations, including Density Functional Theory (DFT) calculations. While it is not open source, it does offer a free version for academic and personal use.
- [Quantum ESPRESSO](https://www.quantum-espresso.org/) - An integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale.

## Molecular dynamics simulations

- [GROMACS](https://www.gromacs.org/) - A versatile package to perform molecular dynamics, scalable and efficient in performing large-scale simulations.
- [LAMMPS](https://www.lammps.org/) - A classical molecular dynamics simulation code, designed to run efficiently on parallel computers.
- [NAMD](https://www.ks.uiuc.edu/Research/namd/) - A parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems.
- [AMBER](https://ambermd.org/) - A suite of biomolecular simulation programs that includes several force fields for the simulation of proteins, nucleic acids, and carbohydrates.
- [Desmond](https://www.deshawresearch.com/resources.html) - Developed by Schrödinger at D. E. Shaw Research, Desmond is a high-performance molecular dynamics simulation program with a focus on drug discovery.

## Topology preparation

- [CGenFF](https://cgenff.umaryland.edu/) - Provides a force field parameterization platform for small organic molecules within the CHARMM force field. It assists researchers in preparing molecular topologies for use in molecular dynamics simulations.
- [SwissParam](https://www.swissparam.ch/) - A tool for generating force field parameters for small molecules. 
- [Automated Topology Builder](https://atb.uq.edu.au/) - A web-based tool for generating molecular topologies and force field parameters.
- [CHARMM-GUI](https://www.charmm-gui.org/) - A user-friendly interface for CHARMM, providing tools for the generation of molecular topologies and input files for simulations.

## Normal mode analysis for predicting protein motions

- [iMod Server](https://imods.iqfr.csic.es/) - A web-based platform dedicated to Normal Mode Analysis (NMA) for predicting protein motions. It allows researchers to analyze and simulate the vibrational modes of proteins, providing valuable insights into their dynamic behavior and structural flexibility. 

## Virtual screening server for drug repurposing

- [DrugRep](http://cao.labshare.cn:10180/DrugRep/php/index.php) - A virtual screening server designed for drug repurposing. 

## Peptide design tools

- [PepDraw](https://pepdraw.com/) - A web-based tool for designing and visualizing peptide structures.
- [PepSite](http://pepsite2.russelllab.org/) - A peptide design tool that aids researchers in identifying potential binding sites on protein surfaces.
- [Peptimap](https://peptimap.bu.edu/) - A tool dedicated to the analysis and visualization of peptide structures.


## PROTAC database and ternary complex modelling

- [PROTAC-db](http://cadd.zju.edu.cn/protacdb/) - A database focused on Proteolysis Targeting Chimeras (PROTACs). It provides a resource for researchers interested in PROTACs by offering information on their design, targets, and associated experimental data, aiding in the exploration of targeted protein degradation.  
- [PROsettaC](https://prosettac.weizmann.ac.il/) - A platform for ternary complex modeling, specifically focusing on the structural modeling of protein-protein interactions. It provides tools for predicting the three-dimensional structures of protein complexes, contributing to the understanding of PROTAC-induced ternary complexes and their implications in drug discovery.

## Machine learning for drug discovery

- [RDKit](https://www.rdkit.org/) - An open-source toolkit for cheminformatics and medicinal chemistry, RDKit facilitates tasks like molecular structure representation, substructure searching, and descriptor calculation.
- [Google Colab](https://colab.research.google.com/) - A cloud-based platform that provides free access to Jupyter notebooks along with GPU support. It allows users to run and share Python code collaboratively, making it particularly useful for data analysis, machine learning, and research projects.
- [Anaconda](https://www.anaconda.com/) - A distribution platform for Python and R programming languages, designed for data science, machine learning, and scientific computing.
- [Pandas](https://pandas.pydata.org/) - An open-source data manipulation and analysis library for Python. It provides easy-to-use data structures, such as dataframes, and a plethora of functions for data cleaning, exploration, and transformation.
- [Numpy](https://numpy.org/) - A library for numerical computing in Python. It provides support for large, multi-dimensional arrays and matrices, along with a collection of mathematical functions to operate on these arrays.
- [Scikit-Learn](https://scikit-learn.org/stable/) - An open-source machine learning library for Python. It offers a simple and efficient toolkit for various machine learning tasks, including classification, regression, clustering, and dimensionality reduction.
- [Matplotlib](https://matplotlib.org/) - A popular 2D plotting library for Python.
- [Seaborn](https://seaborn.pydata.org/) - A statistical data visualization library based on Matplotlib. It simplifies the creation of attractive and informative statistical graphics in Python.
- [MoleculeNet](https://moleculenet.org/) - Offers datasets for benchmarking predictive models of various chemical drug properties.
- [Kaggle](https://www.kaggle.com/) - A large ML and AI community. Can be useful for sourcing and sharing datasets and models related to drug discovery.
- [Hugging Face](https://huggingface.co/) - A new growing AI community similar to Kaggle.
- [Code Ocean](https://codeocean.com/) - A cloud-based platform designed to facilitate the sharing, collaboration, and reproducibility of research code and data.
- [Zenodo](https://zenodo.org/) - An open-access repository for research outputs. It is an initiative developed by CERN to provide a platform for researchers to share and preserve various types of scholarly content, including datasets, software, publications, and other research-related materials.
- [ChemML](https://hachmannlab.github.io/chemml/index.html) - ChemML empowers users to perform various data science tasks and ML workflows, making modern data science accessible in the broader chemistry and materials community.
- [Datagrok]( https://datagrok.ai/cheminformatics) - Datagrok offers robust support for small molecules and popular building blocks in cheminformatics. It understands various chemical notations like SMILES and SMARTS. Users can visualize molecules in 2D or 3D, sketch them, and extract properties.

## Artificial intelligence for drug discovery

- [Keras](https://keras.io/) - An open-source high-level neural networks API written in Python. It provides a user-friendly interface for building and experimenting with deep learning models. 
- [TensorFlow](https://www.tensorflow.org/) - An open-source machine learning framework developed by Google. It offers a comprehensive set of tools for building and deploying machine learning models, with a particular emphasis on deep learning. 
- [DeepChem](https://deepchem.io/) -  An open-source library for deep learning in drug discovery and cheminformatics
- [TorchDrug](https://torchdrug.ai/) - A cheminformatics toolkit based on PyTorch. It provides functionalities for molecular property prediction, compound screening, and deep learning-based analyses.  
- [DEEPScreen](https://github.com/cansyl/DEEPScreen) - A Python-based tool for virtual screening studies with deep convolutional neural networks using compound images.  
- [GraphINVENT](https://github.com/MolecularAI/GraphINVENT) - A Python-based platform deployed by the MolecularAI group at AstraZeneca, for molecular design based on graph generation models. It employs graph neural networks to generate molecular graphs with desired properties.

## Retrosynthesis prediction

- [Spaya AI-powered retrosynthesis platform](https://spaya.ai/app/search) - An AI-powered platform for retrosynthetic analysis. It assists chemists in designing synthetic routes for target compounds.
- [AiZynthFinder](https://github.com/MolecularAI/aizynthfinder) - A tool based on Monte Carlo tree search, designed for retrosynthetic planning in organic chemistry. It aids chemists in generating synthetic routes by exploring reaction databases and proposing viable retrosynthetic steps.
- [ASKCOS](https://askcos.mit.edu/) -  An AI-driven platform developed for computer-assisted organic synthesis planning. Utilizing machine learning algorithms, ASKCOS assists chemists in predicting viable reaction pathways and proposing synthetic routes for target molecules.
- [IBM RoboRXN](https://rxn.res.ibm.com/rxn/robo-rxn/welcome) - An AI-driven platform for automated reaction prediction and planning. Using advanced machine learning models, it enables researchers to predict the outcomes of chemical reactions and design synthetic routes for desired products.
- [MANIFOLD](https://app.postera.ai/manifold/) - Offers tools for synthesis planning, compound search, enabling rapid identification and production of high-quality, synthetically accessible compounds.  


## Miscellaneous tools

- [OPSIN: Open Parser for Systematic IUPAC nomenclature](https://opsin.ch.cam.ac.uk) - Useful for converting chemical names into structured representations.
- [OSRA: Optical Structure Recognition](https://cactus.nci.nih.gov/cgi-bin/osra/index.cgi) - An Optical Structure Recognition tool that converts graphical representations of chemical structures, such as images, into connection tables. 
- [MetaPredict](http://metapredict.icoa.fr/) - A tool designed for predicting various molecular properties and activities. It supports computational chemistry by providing predictive models for diverse chemical entities.   
- [RPBS Web Portal](https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#welcome) - A platform offering a range of tools for computational biology and chemistry.
- [AI based scoring function platform](http://cadd.zju.edu.cn/asfp/?fbclid=IwAR0SlLlG6WxHVj7O3xvTufTWHdQR_heLefAWqGJXfY3z39-0sy3NxTd5X5E) - Employs artificial intelligence to develop scoring functions for evaluating molecular interactions. It aids in predicting binding affinities and guiding drug discovery efforts.
- [ChemPlot: A Tool For Chemical Space Visualization](https://chemplot.streamlit.app/) - A tool dedicated to visualizing chemical space. 
- [ChemDB Chemoinformatics Portal](http://cdb.ics.uci.edu/) - A chemoinformatics portal offering various tools for chemical data analysis. It supports tasks such as compound search, similarity analysis, and property prediction.  
- [Open Targets Platform](https://platform.opentargets.org/) - The Open Targets Platform integrates genetic, genomic, and chemical data for target identification in drug discovery. 
- [Screening Explorer](http://stats.drugdesign.fr/) - A tool designed for compound screening and analysis.
- [LigRMSD](https://ligrmsd.appsbio.utalca.cl/) - A tool for calculating the root-mean-square deviation (RMSD) between ligand structures.  
- [BoBER: web interface to the base of bioisosterically exchangeable replacements](http://bober.insilab.org/) - A web interface providing bioisosteric replacements for chemical structures.
- [Disease List Automatically Derived For You](http://dlad4u.zhang-lab.org/index.php) - This tool automatically generates a list of diseases based on relevant data and criteria.
- [Python Code Examples](https://www.programcreek.com/python/) - Provide code snippets and examples for common tasks in the Python programming language. It's a valuable resource for learning and implementing Python in scientific computing.
- [NERDD New E-Resource for Drug Discovery](https://nerdd.univie.ac.at/) - A new e-resource focused on drug discovery. It provides information and tools to support researchers in the drug development process.
- [MetaChemiBio](https://biochemia.uwm.edu.pl/metachemibio/) - A tool designed for predicting molecular properties and activities. 
- [The Utrecht Biomolecular Interactions software portal](https://wenmr.science.uu.nl/) - This portal provides software tools for studying biomolecular interactions. 
- [LigBuilder3](http://www.pkumdl.cn:8080/ligbuilder3/) - A tool for building ligand structures and conducting virtual screening. 
- [MolAiCal](https://molaical.github.io/) - Useful for binding free energy calculations.
- [ChemMine Tools](https://chemminetools.ucr.edu/) - Provide a collection of cheminformatics and computational chemistry tools.
- [MayaChemTools](http://www.mayachemtools.org/index.html) - A growing collection of Perl and Python scripts, modules, and classes designed to support diverse daily computational discovery needs.
- [SCBDD](http://www.scbdd.com/) - An online platform containing a series of software and web servers that can assist in cheminformatics and drug discovery.
- [Click2Drug](https://www.click2drug.org/) - Deployed by the Swiss Institute of Bioinformatics, presents an extensive compilation of CADD software, databases, and web services. This collection categorizes tools based on their application field, aiming to encompass the entire drug design pipeline.
- [Galaxy Europe](https://usegalaxy-eu.github.io/index-cheminformatics.html) - A Galaxy instance focused on Cheminformatics.
- [CADD Vault](https://drugbud-suite.github.io/CADD_Vault/) - An open-source repository dedicated to sharing resources, tools, and knowledge in the field of computer-aided drug design.
- [BioMoDes](https://abeebyekeen.com/biomodes-biomolecular-structure-prediction/) - A Repository of Tools for Biomolecular Modeling and Design.
## Cheminformatics free courses

- [Computational chemistry lectures by TMP Chem](https://youtube.com/playlist?list=PLm8ZSArAXicIWTHEWgHG5mDr8YbrdcN1K) -  A YouTube playlist offering computational chemistry lectures by TMP Chem. It covers various aspects of computational chemistry, providing valuable insights for learners interested in the field.
- [Strasbourg Summer School in Chemoinformatics, 2022](https://youtube.com/playlist?list=PLhgURFExPmJsDuHevu5n8y0R41WsXfbnC) - A YouTube playlist featuring lectures from the Strasbourg Summer School in Chemoinformatics. The content covers topics related to cheminformatics.
- [BIGCHEM](https://bigchem.eu/node/63) - The BIGCHEM project provides free courses on big data in chemistry. It offers online resources and training materials to explore the intersection of big data and chemistry, fostering understanding and skills in cheminformatics.
- [Geometric Deep Learning Course](https://geometricdeeplearning.com/lectures/) - A course on Geometric Deep Learing delivered as part of the African Master’s in Machine Intelligence.
- [Drug Discovery Course by StereoElectronics](https://www.stereoelectronics.org/webDD/DD_home.html) - Fundamentals and principles of methods used in the drug discovery pipeline.
- [drugdesign.org](https://www.drugdesign.org/) - A collection of free courses on drug design, cheminformatics, molecular modeling, property prediction, and QSAR.
- [PlayMolecule](https://open.playmolecule.org/landing) -  It focuses on drug discovery and molecular modeling, allowing users to simulate and visualize molecular interactions and properties efficiently.  

## Blogs

- [Practical Fragments](http://practicalfragments.blogspot.com/) - A blog focusing on fragment-based drug discovery. It provides practical insights, case studies, and discussions on the use of small fragments in drug design.
- [avrilomics](https://avrilomics.blogspot.com/) - A blog covering topics in genomics, bioinformatics, and related fields. It offers insights, updates, and discussions on advancements and practical applications in these areas.
- [Practical Cheminformatics](http://practicalcheminformatics.blogspot.com/) - A blog dedicated to practical aspects of cheminformatics. It explores tools, techniques, and applications in the field of chemical informatics and computational chemistry.
- [Cheminformania](https://www.cheminformania.com/) - A compilation of articles focused on valuable insights and hands-on demonstrations detailing diverse cheminformatic tasks, delving progressively deeper into the realm of deep learning and AI applications.
- [Daily Dose of Data Science](https://www.blog.dailydoseofds.com/) - A blog that consolidates captivating frameworks, libraries, technologies, and insights to streamline the entire process of a Data Science project.
- [Machine Learning Mastery](https://machinelearningmastery.com/) - A valuable online resource that offers tutorials, guides, and practical insights into machine learning concepts, algorithms, and techniques. Authored by Jason Brownlee, a machine learning expert and educator, the site covers a wide range of topics, from fundamental concepts for beginners to advanced techniques for seasoned practitioners.
- [Chem-Workflows](https://chem-workflows.com/index.html) - A collection of Jupyter Notebook-based tutorials written by Dr. Angel J. Ruiz Moreno for chemical data exploration and visualization.
- [Structural Bioinformatics](https://proteinstructures.com/) - A guide to structural biology and structure-based drug design.
- [Bioinformatics Answers](https://www.biostars.org/) - An online platform designed for bioinformatics researchers, computational biologists, and individuals involved in life sciences. It serves as a community-driven question and answer platform, where users can seek help, share knowledge, and engage in discussions related to bioinformatics and computational biology.
- [McConnellsMedChem](https://mcconnellsmedchem.com/) - A Medicinal chemistry blog serving as a resource for professionals, researchers, or students involved in the field of medicinal chemistry.  
- [DrugDiscovery.NET](http://www.drugdiscovery.net/) - A blog about AI and machine learning in drug discovery.  
- [MacinChem](https://macinchem.org/) - A blog covering computational chemistry, cheminformatics, and molecular modeling on macOS.  

